plugins/kubernetes-backend/src/service/KubernetesBuilder.ts

```diff
@@ -325,4 +340,28 @@ export class KubernetesBuilder {
 
     return objectTypesToFetch;
   }
+
+  protected async makeProxyRequest(
+    req: express.Request,
+    res: express.Response,
+  ) {
+    const services = this.getProxyServices();
+    const proxyResponse: KubernetesProxyResponse =
+      await this.proxy.handleProxyRequest(services, req);
+    res.status(proxyResponse.code).json(proxyResponse.data);
+  }
+
+  protected getProxyServices(): KubernetesProxyServices {
```

Comment:
Both of these new methods be private? Then you wouldn't have to make the KubernetesProxyServices type public

Reply:
Done